sm/plugins/tedge_apt_plugin/src/module_check.rs

```diff
@@ -0,0 +1,72 @@
+use crate::error::InternalError;
+use std::path::PathBuf;
+use std::process::{Command, Stdio};
+
+/// check that module_name is in file path
```

Comment:
The comment is misleading. 

```suggestion
/// check that the path of the package to install is actually shaped as expected by apt
```